code/modules/lighting/lighting_corner.dm

```diff
@@ -83,11 +83,11 @@ GLOBAL_LIST_INIT(LIGHTING_CORNER_DIAGONAL, list(NORTHEAST, SOUTHEAST, SOUTHWEST,
 		T = thing
 		if (T.lighting_object)
 			active = TRUE
+			return
 
 // God that was a mess, now to do the rest of the corner code! Hooray!
 /datum/lighting_corner/proc/update_lumcount(delta_r, delta_g, delta_b)
-
-	if ((abs(delta_r)+abs(delta_g)+abs(delta_b)) == 0)
```

Comment:
what the fuck would this even do lol the absolute value of 0 is 0. what were they thinking?

Reply:
They were probably thinking that `x + y = 0` when `x = -y` and that for some reason the comparison operation would be more costly than the math ones.

Reply:
the original version was `x+y`, when that caused a bug it was changed to `abs(x)...` because that was likely what seemed the logical fix to the problem of negative values.